### PR TITLE
Create calculator component for the app

### PR DESCRIPTION
**Pull request by JonahKayizzi**
This pull request consists of the following changes;

- Deleted all the boilerplate from CRA (text, images, styles).
- Created a new Calculator component.
- Styled the calculator as per the provided design

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,7 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return <Calculator />;
 }
```
Comment (mirouhml) [inline src/App.js]: - [ ] Please know that the App.js component should be class-based, kindly fix this.

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @JonahKayizzi, 

Good job so far!

There is only one issue that you still need to work on to go to the next project but you are almost there!



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (JonahKayizzi) [reply src/App.js]: > * [ ]  Please know that the App.js component should be class-based, kindly fix this.

Let me fix this. Thanks for the advice

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @JonahKayizzi , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

### Highlights:
 ✔️ Project is setup correctly
 ✔️ All linters are passing
✔️ Correct gitflow
 ✔️ Professional documentation

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

image

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@shoirata] in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**